portfolio/src/main/webapp/script.js

```diff
@@ -165,3 +166,13 @@ function createCommentElement(comment) {
 
   return commentElement;
 }
+
+/**
+ * Delete all comments from datastore.
+ */
+function deleteAllComments() {
+  fetch('/delete-data', {
```

Comment:
this is very risky since an accidental click can delete all the comments. You should try to add simple confirm dialog to avoid this. See https://www.w3schools.com/js/js_popup.asp for some options.